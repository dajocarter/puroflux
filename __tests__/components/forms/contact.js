import React from 'react'
import renderer from 'react-test-renderer'

import ContactForm from '../../../src/components/forms/contact'

describe('Contact Form', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ContactForm />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
