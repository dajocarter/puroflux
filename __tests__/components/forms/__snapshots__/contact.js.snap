// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Form renders correctly 1`] = `
<form
  data-netlify="true"
  data-netlify-honeypot="bot-field"
  method="POST"
  name="Contact Us"
>
  <input
    name="form-name"
    type="hidden"
    value="Contact Us"
  />
  <p
    className="d-none"
  >
    <label>
      Donâ€™t fill this out:
       
      <input
        name="bot-field"
        type="text"
      />
    </label>
  </p>
  <div
    className="form-group"
  >
    <label
      className="form-label"
      htmlFor="name"
    >
      Name
    </label>
    <input
      className="form-control"
      id="name"
      name="name"
      placeholder="Your name"
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label
      className="form-label"
      htmlFor="email"
    >
      Email
    </label>
    <input
      className="form-control"
      id="email"
      name="_replyto"
      placeholder="Your email"
      type="email"
    />
  </div>
  <div
    className="form-group"
  >
    <label
      className="form-label"
      htmlFor="message"
    >
      Message
    </label>
    <textarea
      className="form-control"
      id="message"
      name="message"
      placeholder="Your message"
    />
  </div>
  <div
    className="form-group"
  >
    <label
      className="form-label"
      htmlFor="location"
    >
      Location
    </label>
    <input
      className="form-control"
      id="location"
      name="location"
      placeholder="Your location"
      type="text"
    />
  </div>
  <input
    name="_subject"
    type="hidden"
    value="New submission from Contact Us form."
  />
  <button
    className="btn btn-light"
    disabled={false}
    type="submit"
  >
    Submit
  </button>
</form>
`;
